diff --git a/board/at91sam9x5_aria/at91sam9x5_aria.c b/board/at91sam9x5_aria/at91sam9x5_aria.c
index ffcf6ce..ea95998 100644
--- a/board/at91sam9x5_aria/at91sam9x5_aria.c
+++ b/board/at91sam9x5_aria/at91sam9x5_aria.c
@@ -238,7 +238,7 @@ static void sdcard_set_of_name_board(char *of_name)
 {
 	//unsigned int cpu_board_id = get_cm_sn();
 	//unsigned int disp_board_id = get_dm_sn();
-	strcpy(of_name, "at91-ariag25");
+	strcpy(of_name, "acme-aria");
 	strcat(of_name, ".dtb");
 }
 
